lib/zendesk_apps_tools/command.rb

```diff
@@ -162,11 +162,14 @@ def create
         app_name = manifest.name
       end
       app_name ||= get_value_from_stdin('Enter app name:')
-      deploy_app(:post, '/api/v2/apps.json', name: app_name)
+      deploy_app(:post, '/api/apps.json', name: app_name)
```

Comment:
Will probably need to update [this spec](https://github.com/zendesk/zendesk_apps_tools/blob/master/spec/lib/zendesk_apps_tools/command_spec.rb#L60).